_workinggroups/quantumcomputing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Quantum Computing"
+author: "Riccardo Maria Bianchi"
+layout: plain
+---
+
+
+The Quantum Computing Working Group fosters collaboration among the HEP experiments and institutes to exploit the possibilities offered by quantum computing.
+
+Quantum computing is a very active research field and prototype computers, simulators, and toolkits are now available. The participants in the working group share experiences, ideas, know-how, demonstrators, code snippets, and documents with the aim of fostering common research and development in this field, for the benefit of the HEP community.
+
+Kick-starters: *Riccardo Maria BIANCHI* (ATLAS), *Sebastien BINET* (LSST, ALICE, LPC), *Riccardo DI SIPIO* (ATLAS), *Lukas HEINRICH* (ATLAS), *Maurizio PIERINI* (CMS)
+
+## Contacts
+
+- Convenors: [hsf-quantumcomputing-wg-convenors@googlegroups.com](mailto:hsf-quantumcomputing-wg-convenors@googlegroups.com)
+- Working group's mailing list: [hsf-quantumcomputing-wg@googlegroups.com](hsf-quantumcomputing-wg@googlegroups.com)
```

Comment:
Ah! Last mistake - this should be a `mailto:` link.

Reply:
Yes! Good catch!
Thanks!

Now, fixed.